src/WebJobs.Extensions.Twilio/TwilioSMSAttribute.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.WebJobs
+{
+    /// <summary>
+    /// Attribute used to bind a parameter to a Twilio SMSMessage that will automatically be
+    /// sent when the function completes.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Parameter)]
+    public sealed class TwilioSmsAttribute : Attribute
```

Comment:
>TwilioSmsAttribute [](start = 24, length = 18)

You're going to need to support AccountSid and AuthToken here as well so you can set them on each Script. 

AuthToken will definitely be one of the 'secret' properties that you'll want to require to be an app setting (resolved by the NameResolver). I'm not sure about AccountSid.